src/main/java/core/basesyntax/GeometricFigureSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class GeometricFigureSupplier {
```

Comment:
The class name 'GeometricFigureSupplier' implies that it supplies geometric figures, but it seems to be supplying enum constants. Consider renaming the class to reflect its purpose more accurately, such as 'GeometricFigureTypeSupplier', if it's supplying types of figures.